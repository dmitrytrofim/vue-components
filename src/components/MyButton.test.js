import { mount } from '@vue/test-utils';
import MyButton from './MyButton.vue';

test('increments count on click', async () => {
  const wrapper = mount(MyButton);
  expect(wrapper.text()).toBe('0');

  await wrapper.find('button').trigger('click');
  expect(wrapper.text()).toBe('1');

  await wrapper.find('button').trigger('click');
  expect(wrapper.text()).toBe('2');

  expect(wrapper.html()).toMatchSnapshot();
});
