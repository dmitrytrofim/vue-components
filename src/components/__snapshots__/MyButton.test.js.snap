// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`increments count on click 1`] = `"<button>2</button>"`;
